doc/tools/dorado-scheduling-animate-survey.rst

```diff
@@ -0,0 +1,5 @@
+Animate a full survey simulation (``dorado-scheduling-animate-survey``)
+=========================================================
```

Comment:
The underline made of `=` signs needs to be the same length as the heading above it.